tests/test_custom_checkers/test_invalid_name_checker.py

```diff
@@ -0,0 +1,354 @@
+"""Test suite for testing the InvalidNameChecker."""
+import sys
+import unittest
+
+import astroid
+import pylint.testutils
+from astroid import nodes
+
+from python_ta.checkers.invalid_name_checker import InvalidNameChecker
+
+
+class TestInvalidNameChecker(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = InvalidNameChecker
+    CONFIG = {
```

Comment:
This points to a deeper problem. You shouldn't have to add a config option that isn't used anywhere in the class; find a better solution.

Reply:
Removed the function that would cause the error so this shouldn't be a problem anymore